src/__tests__/getTemplateLocation.test.ts

```diff
@@ -13,10 +20,12 @@ describe("getTemplateLocation", () => {
     expect(templateLocation).toEqual(DEFAULT_TEMPLATE_PATH);
   });
 
-  it.skip("uses developer custom templates if defined", () => {
+  it("uses developer custom templates if defined", () => {
     jest.mock("fs");
     jest.spyOn(process, "cwd").mockImplementation(() => "/custom/path/to");
 
+    const { getTemplateLocation } = require("../utils/getTemplateLocation");
```

Comment:
Why's this getting reimported?

Reply:
Yes! Great question. I asked the same thing (Alvin wrote the original test).

The reason for that is this: we're spying on `process.cwd()`, which is used inside the `getTemplateLocation` function's file. By reimporting it, within the `it` block after we've mocked the implementation for `process.cwd`, the mock works as expected and the test assertion passes.

If we remove that and use the "global" import for that file, `process.cwd()` does not use our mock. I found this out through trial and error, and from reading these relevant Jest issues:

- [process.cwd is not mocked for path.resolve](https://github.com/facebook/jest/issues/5146)
- [Jest globals differ from Node globals](https://github.com/facebook/jest/issues/2549)

I will add a comment to explain that (condensed of course). Great question!